addon/components/imgix-image.js

```diff
@@ -339,6 +341,10 @@ export default Component.extend({
     return placeholderURL;
   }),
 
+  elementClassNames: computed('config.APP.imgix.classNames', function() {
+    return config.APP.imgix.classNames || 'imgix-image';
```

Comment:
Would there ever be a case where someone would want *no* default class on the img element? This `|| 'imgix-image'` would make it impossible to unset this option entirely.